modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm_{context}']
+ifdef::virt-importing-vmware-vm[]
+= Importing a VMware virtual machine or template
+
+You can import a VMware virtual machine or template using the virtual machine wizard.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+= Importing a Red Hat Virtualization virtual machine with the virtual machine wizard
+
+You can import a Red Hat Virtualization (RHV) virtual machine using the virtual machine wizard.
+endif::[]
+
+.Procedure
+
+. In the web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create Virtual Machine* and select *Import with Wizard*.
+
+ifdef::virt-importing-vmware-vm[]
+. In the *Connect to Provider* screen, select *VMware* from the *Provider* list.
+. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
+
+* If you select *Connect to New Instance*, enter the *vCenter hostname*, *Username*, and *Password*.
+* If you select a saved vCenter instance, the wizard connects to the vCenter instance using the saved credentials.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+. In the *Connect to Provider* screen, select *Red Hat Virtualization (RHV)* from the *Provider* list.
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
```

Comment:
1. I would remove for example. If this is not entered in this format it will not work
2. When I tested it, I did not add the :8443

Reply:
OK, I'll remove the port